src/cli.js

```diff
@@ -50,7 +53,7 @@ let nodeModuleVersion = null;
 if (!argv.n) {
   try {
     let pathDotText = path.join(argv.e, 'path.txt');
-    electronPath = fs.readFileSync(pathDotText, 'utf8');
+    electronPath = path.join(argv.e, fs.readFileSync(pathDotText, 'utf8'));
```

Comment:
You should use path.resolve instead of path.join, so it can handles both cases, when path.txt contains absolute or relative path.